src/UI/Implementation/Component/Deck/Deck.php

```diff
@@ -55,7 +55,49 @@ public function getCards(){
 	/**
 	 * @inheritdoc
 	 */
-	public function withCardsSize($size){
+	public function withExtraSmallCards(){
+		return $this->withCardsSize(self::SIZE_XS);
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function withSmallCards(){
+		return $this->withCardsSize(self::SIZE_S);
+	}
+	/**
+	 * @inheritdoc
+	 */
+	public function withNormalCards(){
+		return $this->withCardsSize(self::SIZE_M);
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function withLargeCards(){
+		return $this->withCardsSize(self::SIZE_L);
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function withExtraLargeCards(){
+		return $this->withCardsSize(self::SIZE_XL);
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	public function withFullSizedCards(){
+		return $this->withCardsSize(self::SIZE_FULL);
+	}
+
+	/***
+	 * @param $size
+	 * @return Deck
+	 */
+	protected function withCardsSize($size){
```

Comment:
I like this change, although passing class constants is absolutely fine for me, as long as other contexts are not scattered with these constants as dependencies.